lua/gatekeeper.lua

```diff
@@ -99,6 +99,19 @@ local ffi = require("ffi")
 -- Structs
 ffi.cdef[[
 
+enum gk_flow_state {
```

Comment:
I think that everything that is being added to `lua/gatekeeper.lua` in this patch should go to a new file: `policylib.lua`. The `gatekeeper.lua` is used to help Gatekeeper to boot, and `policylib.lua` to help write policies.